src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleTypeSymbol.vb

```diff
@@ -0,0 +1,987 @@
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsComImport() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsWindowsRuntimeImport() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property ShouldAddWinRTMembers() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsSerializable() As Boolean
+            Get
+                Return Me._underlyingType.IsSerializable
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property Layout() As TypeLayout
+            Get
+                Return Me._underlyingType.Layout
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property MarshallingCharSet() As CharSet
+            Get
+                Return Me._underlyingType.MarshallingCharSet
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property HasDeclarativeSecurity() As Boolean
+            Get
+                Return Me._underlyingType.HasDeclarativeSecurity
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsInterface() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property IsExtensibleInterfaceNoUseSiteDiagnostics As Boolean
+            Get
+                Return Me._underlyingType.IsExtensibleInterfaceNoUseSiteDiagnostics
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property CanConstruct As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Friend Overrides ReadOnly Property TypeSubstitution As TypeSubstitution
+            Get
+                Return Nothing
+            End Get
+        End Property
+
+        Private Sub New(locationOpt As Location, underlyingType As NamedTypeSymbol, elementLocations As ImmutableArray(Of Location), elementNames As ImmutableArray(Of String), elementTypes As ImmutableArray(Of TypeSymbol))
+            Me.New(If((locationOpt Is Nothing), ImmutableArray(Of Location).Empty, ImmutableArray.Create(Of Location)(locationOpt)), underlyingType, elementLocations, elementNames, elementTypes)
+        End Sub
+
+        Private Sub New(locations As ImmutableArray(Of Location), underlyingType As NamedTypeSymbol, elementLocations As ImmutableArray(Of Location), elementNames As ImmutableArray(Of String), elementTypes As ImmutableArray(Of TypeSymbol))
+            MyBase.New(underlyingType)
+            Debug.Assert(elementLocations.IsDefault OrElse elementNames.IsDefault OrElse elementLocations.Length = elementNames.Length)
```

Comment:
Fixed in both C# and VB

---
In reply to: [71554276](https://github.com/dotnet/roslyn/pull/12596#discussion_r71554276) [](ancestors = 71554276)